modules/git/repo_commit.go

```diff
@@ -283,7 +283,7 @@ func (repo *Repository) CommitsBetween(last, before *Commit) ([]*Commit, error)
 	return repo.parsePrettyFormatLogToList(bytes.TrimSpace(stdout))
 }
 
-// CommitsBetweenLimit returns a list that contains at most limit commits skipping the first skip commits between [before, last)
+// CommitsBetweenLimit returns a list that contains at most limit commits skipping the first skip commits between [before, last]
```

Comment:
Can we have a more clear comment about `[before, last)` or `[before, last]`? Is `last` included or excluded?

Reply:
I think that was exactly the reason why `[before, last)` has been changed: Because the documentation was previously erroneously in that it implied that `last` would be excluded from the returned list:
I interpret the parentheses as interval notation.
What is however not obvious is that `before` can be `nil` to search for all previous commits. That should be documented as well.